core/src/streams.rs

```diff
@@ -218,6 +218,8 @@ pub struct NetStreamData<'gc> {
     /// Seeks are only executed on the next stream tick.
     queued_seek_time: Option<f64>,
 
+    buffer_time: f64,
```

Comment:
can you also document it as todo-currently-unused?

Reply:
> can you also document it as todo-currently-unused?

I think it requires buffer_time